### ISSUE TITLE
Add option for cabin calls/seatbelts/no smoking without being muted by the closed door

### ISSUE THREAD

**Issue by ErikNL80** [flybywiresim/a32nx#5696]
### Aircraft Version

Development

### Description

In the latest dev sounds are updated in a way that if you close the cabin door you will not hear the seatbelt/no smoking chimes and cabin call sounds. Although this is realistic behaviour I would like to request an option to the EFB to override this behaviour (without having to leave the door open)... (See https://github.com/flybywiresim/a32nx/issues/5695)

This could be done by using the existing option of enabling the PTU sound... If you have enabled this option the seatbelt/no smoking/cabin chimes will sound 'unrealistic'. 

So in short you could make 1 setting 'unrealistic sounds' which covers all?

A lot of devs are struggling between how an aircraft really sounds and what flightsim users expect to hear (since they always travel in the cabin) :) I think you make a lot of people happy with this. 

Thanks for considering!

**Comment by hotshotp** [comment]
Hi! I am the one who made the change with the door. We will not make an option for such a niche thing, as it takes up valuable space in the efb settings page, and it is just unnecessary. Nothing is preventing you from flying with the door open!

**Comment by ErikNL80** [comment]
Hi, thanks for the response... As I suggested above you can use an existing option (for the PTU sound) and add the others here as well...

**Comment by ZigTag** [comment]
While it's possible to add, the feature is not in demand, so we are not going to implement it

**Comment by ErikNL80** [comment]
How do you now if the issue is closed within 30 minutes?

**Comment by sidnov** [comment]
Reopening for feedback from other users (if any)

**Comment by ErikNL80** [comment]
Thanks sidnov! I understand it is a low prio issue compared to other stuff...

**Comment by hotshotp** [comment]
> Hi, thanks for the response... As I suggested above you can use an existing option (for the PTU sound) and add the others here as well...

again, its a very niche thing, that we do not need to add. Again, just open the door. I am not altering the sound set so much that there is an "unrealistic" and a "realistic" option. We are an addon that strives to be as realistic as possible, and we only added the PTU option because it was in very high demand.

**Comment by hotshotp** [comment]
Closing due to lack of responses, and overall an un-needed feature request.
